sources/deeptree/src/counters.js

```diff
@@ -1,28 +1,35 @@
-import {createSlice} from "redux-starter-kit";
+import { createSlice } from "redux-starter-kit";
 import * as c from "./constants";
 
-const {reducer, actions} = createSlice({
-    initialState : {},
-    reducers : {
-        initialize(state, action) {
-            const {numberOfCounters} = action.payload;
-            for(let i = 0; i < numberOfCounters; i++) {
-                state[i] = 0;
-            }
-        },
-        increment(state, action) {
-            const {counterId} = action.payload;
-            const value = state[counterId] || 0;
-            state[counterId] = value + 1;
+const { reducer, actions } = createSlice({
+  initialState: {},
+  reducers: {
+    initialize(state, action) {
+      const { numberOfCounters } = action.payload;
+      for (let i = 0; i < numberOfCounters; i++) {
+        state[i] = 0;
+      }
+    },
+    increment(state, action) {
+      const { counterId } = action.payload;
```

Comment:
As `incrementMany` is the thing, it's not important, but you are missing:

```
const value = state[counterId] || 0;
state[counterId] = value + 1;
```

Reply:
Whoops, looks like I broke this while making edits!  Good catch, thanks.